locale/ua.json

```diff
@@ -0,0 +1,3171 @@
+                "whatCanActuallyDo": "Що насправді може робити {{shortTitle}}? І як я можу використовувати його для організації, оформлення та проведення своїх уроків? Перегляньте наші",
+                "youAreAdminAndHaveTheHonorable": "Ви адміністратор і маєте почесне завдання встановити {{title}} для вашої школи? Ми допоможемо вам у цій важливій справі. Перегляньте наші"
+            }
+        },
+        "welcome": {
+            "button": {
+                "toYourCourses": "На ваші курси"
+            },
+            "headline": {
+                "congratulations": "Вітаємо, {{firstName}} {{lastName}}",
+                "welcomeToThe": "Ласкаво просимо до {{shortTitle}}{{firstName}}{{lastName}}"
+            },
+            "text": {
+                "dueToAChangeInTheLowerSaxony": "У зв’язку зі зміною в Законі про школи Нижньої Саксонії, обробка персональних даних у {{shortTitle}} тепер в основному базується на § 31 (5) NSchG у поєднанні зі статтею 6, параграф 1 літ. с) GDPR.",
+                "pleaseBrieflyFileSomeData": "Коротко введіть деякі дані, встановіть новий пароль, а потім можете почати.",
+                "thankYouForTakingNoteOf": "Дякуємо, що взяли до відома.",
+                "weArePleasedThatYou": "Нам приємно, що ви хочете використовувати {{shortTitle}}.",
+                "weArePleasedThatYouWantToUse": "Нам приємно, що ви хочете використовувати {{title}} та співпрацювати, щоб зробити наступний крок разом для викладання з цифровою підтримкою — з відкритим кодом, некомерційною і відповідно до правил захисту даних. Перевірте адресу електронної пошти, яку ви вказали в наступних кроках, і погодьтеся з умовами використання. Один з батьків або законний опікун вже дали згоду на його використання, тепер вирішувати вам. Потім змініть початковий пароль, щоб скористатися {{shortTitle}}.",
+                "welcomeBack": "Ласкаво просимо назад до {{shortTitle}} {{firstName}} {{lastName}}",
+                "welcomeTo": "Ласкаво просимо до {{title}} {{displayName}}",
+                "welcomeToShort": "Ласкаво просимо до {{shortTitle}}",
+                "withThe": "Перетворення викладання в цифрове за допомогою {{shortTitle}}",
+                "youAreNow14YearsOld": "Вам зараз 14 років. На наступному кроці надайте свою згоду на використання {{title}} З 14 років це необхідно на додаток до вже наданої згоди ваших батьків, щоб ви могли продовжувати використовувати {{shortTitle}}."
+            }
+        }
+    },
+    "format": {
+        "date": "ММ/ДД/РРРР",
+        "dateLong": "дддд, дд. ММММ РРРР",
+        "datePicker": "м/д/Р",
+        "datePickerMask": "19/39/9999",
+        "dateTime": "ММ/ДД/РРРР ГГ:хх",
+        "dateTimePicker": "м/д/Р Г:х",
+        "dateTimePickerMask": "19/39/9999 29:59",
+        "dateTimeToPicker": "ММ/ДД/РРРР ГГ:хх",
+        "dateToPicker": "ММ/ДД/РРРР"
+    },
```

Comment:
```suggestion
    "format": {
		"date": "DD/MM/YYYY",
		"dateLong": "dddd, DD/ MMMM YYYY",
		"datePicker": "d/m/Y",
		"datePickerMask": "39/19/9999",
		"dateTime": "DD/MM/YYYY HH:mm",
		"dateTimePicker": "d/m/Y H:i",
		"dateTimePickerMask": "39/19/9999 29:59",
		"dateTimeToPicker": "DD/MM/YYYY HH:mm",
		"dateToPicker": "DD/MM/YYYY"
    },
```

Reply:
and wrong foulder locale/ua.json => locales/ua.json

Reply:
DD:MM:YYYYY